nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/nuxeo.defaults

```diff
@@ -155,7 +155,7 @@ elasticsearch.index.translog.durability=request
 # For the rest client, a timeout in milliseconds until a connection is established
 elasticsearch.restClient.connectionTimeoutMs=30000
 # For the rest client, a maximum period, in milliseconds, of inactivity between two consecutive data packets
-elasticsearch.restClient.socketTimeoutMs=20000
+elasticsearch.restClient.socketTimeoutMs=121000
```

Comment:
just for my personal knowledge, is 121k referring to a known value? Or is it just a number to have a lot of time?

Reply:
121 seconds is 2min and 1 second, so the request timeout (which is 2min) could be raised before the socket timeout.